libmozevent/storage.py

```diff
@@ -38,7 +38,6 @@ async def set(self, key, value):
                 await redis.set(
                     self._redis_key(key),
                     pickle.dumps(value, protocol=pickle.HIGHEST_PROTOCOL),
-                    expire=self.expiration,
```

Comment:
The parameter was renamed to `ex`: https://aioredis.readthedocs.io/en/latest/api/high-level/#aioredis.client.Redis.set.